recipes/arb-bio/6.1.rev16223/make.patch

```diff
@@ -0,0 +1,25 @@
+--- Makefile.orig	2017-03-23 18:26:58.000000000 -0600
++++ Makefile	2017-03-23 19:01:36.000000000 -0600
+@@ -213,9 +213,9 @@
+ #----------------------
+
+ ifeq ($(DARWIN),1)
+- LINK_STATIC=1# link static
++ LINK_STATIC?=1# link static
+ else
+- LINK_STATIC=0# link dynamically
++ LINK_STATIC?=0# link dynamically
+ #  LINK_STATIC=1# link static (testing only)
+ endif
+
+@@ -770,6 +770,10 @@
+
+ #---------------------- differences between linking executables and shared libs:
+
++cflags += $(CFLAGS)
++clflags += $(LDFLAGS)
++cxxflags += $(CXXFLAGS)
```

Comment:
Upstream likes to have the build as isolated as possible. I may be able to push a variant of this through, but it won't be fast or easy.